IDEAS/BoundaryConditions/Occupants/Extern/StROBe.mo

```diff
@@ -16,27 +16,30 @@ protected
 
   Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow[nZones] QCon_flow
     annotation (Placement(transformation(extent={{-170,10},{-190,30}})));
-  Modelica.Blocks.Sources.RealExpression[nZones] QCon(y=fV*strobe.tabQCon.y[id])
+  Modelica.Blocks.Sources.RealExpression[nZones] QCon(
+    y=fV*strobe.tabQCon_internal[id])
     annotation (Placement(transformation(extent={{-120,10},{-160,30}})));
   Modelica.Thermal.HeatTransfer.Sources.PrescribedHeatFlow[nZones] QRad_flow
     annotation (Placement(transformation(extent={{-170,-30},{-190,-10}})));
-  Modelica.Blocks.Sources.RealExpression[nZones] QRad(y=fV*strobe.tabQRad.y[id])
+  Modelica.Blocks.Sources.RealExpression[nZones] QRad(
+    y=fV*strobe.tabQRad_internal[id])
     annotation (Placement(transformation(extent={{-120,-30},{-160,-10}})));
-  Modelica.Blocks.Sources.RealExpression mDHW(y=strobe.tabDHW.y[id]/60)
+  Modelica.Blocks.Sources.RealExpression mDHW(y=strobe.tabDHW_internal[id]/60)
```

Comment:
Concerning `mDHW` I think a larger update is needed.
StROBe actually outputs liters/minute water withdrawal. Here there is a division by 60 to make it in l/s. We need to see what would be the best units here. Would SI not be m3/s? Also, it's not really "mass flow". A big discussion, I suggest to open in another issue.

Reply:
SI is kg/s. Best to discuss this in a separate issue indeed, if it's really an issue :)